src/components/Cards/SummaryCard/index.vue

```diff
@@ -105,7 +110,7 @@ export default {
     color: var(--color-text-primary);
     cursor: pointer;
 
-    &:hover {
+    &.can-direct:hover {
       color: var(--color-primary);
     }
   }
```

Comment:
There are no significant code differences between the two files provided. Please ensure you have run any required async functions properly to avoid unexpected behavior or errors when interacting with the elements in this component.

Here's an example of such a function correctly implemented:

```python
async def func_name(whatever):
    ...

```

This should be incorporated into where it was called elsewhere within your codebase so that it performs its intended purpose without causing issues.